packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart

```diff
@@ -165,15 +163,17 @@ flutter:
       );
     });
 
-    testWithoutContext('Only images in folders named with device pixel ratios (e.g. 2x, 3.0x) should be considered as variants of other images', () async {
-      const String image = 'assets/image.jpg';
-      const String image2xVariant = 'assets/2x/image.jpg';
-      const String imageNonVariant = 'assets/notAVariant/image.jpg';
+    testWithoutContext('Variant detection works with windows-style filepaths', () async {
```

Comment:
Aren't these posix/unix style paths?

Reply:
Yeah, I think I went autopilot here. I'll get that fixed.

For context, I decided to consolidate the windows tests into one, so that we wouldn't have *N* duplicated tests.

Reply:
On second reading, I actually noticed that the test actually does use windows-style paths. Check out the use of `correctPathSeperators`

https://github.com/flutter/flutter/blob/9ace46137c4909effec401b49fb642e44d4cc15f/packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart#L180

However, this convo has pointed out that this is confusing, so I reversed the paradigm here, so-to-speak. By this I mean I declared the paths as windows paths and then convert them back to posix for the assertions.